airflow/providers/apache/kylin/hooks/kylin.py

```diff
@@ -0,0 +1,67 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from typing import Optional
+from airflow.exceptions import AirflowException
+from airflow.hooks.base_hook import BaseHook
+from kylinpy import kylinpy
+from kylinpy import exceptions
+
+
+class KylinHook(BaseHook):
+    """
+    :param kylin_conn_id: The connection id as configured in Airflow administration.
+    :type kylin_conn_id: str
+    :param project: porject name
+    :type project: Optional[str]
+    :param dsn: dsn
+    :type dsn: Optional[str]
+    """
+    def __init__(self,
+                 kylin_conn_id: Optional[str] = 'kylin_default',
+                 project: Optional[str] = None,
+                 dsn: Optional[str] = None,
+                 *args,
+                 **kwargs
+                 ):
+        super().__init__()
+        self.kylin_conn_id = kylin_conn_id
+        self.project = project
+        self.dsn = dsn
+        self.invoke_command = kylinpy.CubeSource.support_invoke_command
```

Comment:
Is it used somewhere? What does it do? Could you add a comment?

Reply:
It is used by kylinoperator, I hive moved it to kylin operator